docs/index.md

```diff
@@ -9,6 +9,48 @@
 
 All of the Helm Classic charts live at [github.com/helm/charts](https://github.com/helm/charts). If you want to make your own charts we have a guide for [authoring charts](authoring_charts.md) as well.
 
+## Installing Helm Classic
+
+From a Linux or Mac OS X client:
+```
+curl -s https://get.helm.sh | bash
+```
+
+*or*:
+
+1. Grab a prebuilt binary from:
+  - the latest release: [ ![Download](https://api.bintray.com/packages/deis/helm/helm-classic/images/download.svg) ](https://bintray.com/deis/helm/helm-classic/_latestVersion#files)
+  - the CI build pipeline: [ ![Download](https://api.bintray.com/packages/deis/helm-ci/helm-classic/images/download.svg) ](https://bintray.com/deis/helm-ci/helm-classic/_latestVersion#files)
```

Comment:
the bintray links should have /helmc/_latestVersion instead of helm-classic/_latestVersion because our packages are named so